File: Backend/knowledge-store/code/langchain/agents-intro.ts
// --- Google ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "google-genai:gemini-3.6-flash", tools });

// --- OpenAI ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "openai:gpt-5.5", tools });

// --- Anthropic ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "anthropic:claude-sonnet-4-6", tools });

// --- OpenRouter ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "openrouter:openrouter:z-ai/glm-5.2", tools });

// --- Fireworks ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "fireworks:accounts/fireworks/models/glm-5p2", tools });

// --- Baseten ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "baseten:zai-org/GLM-5.2", tools });

// --- Ollama ---
import { createAgent } from "langchain";

var agent = createAgent({ model: "ollama:north-mini-code-1.0", tools });

File: Backend/knowledge-store/code/langchain/agents-intro.py
# --- Google ---
from langchain.agents import create_agent

agent = create_agent(model="google_genai:gemini-3.6-flash", tools=tools)

# --- OpenAI ---
from langchain.agents import create_agent

agent = create_agent(model="openai:gpt-5.5", tools=tools)

# --- Anthropic ---
from langchain.agents import create_agent

agent = create_agent(model="anthropic:claude-sonnet-4-6", tools=tools)

# --- OpenRouter ---
from langchain.agents import create_agent

agent = create_agent(model="openrouter:z-ai/glm-5.2", tools=tools)

# --- Fireworks ---
from langchain.agents import create_agent

agent = create_agent(model="fireworks:accounts/fireworks/models/glm-5p2", tools=tools)

# --- Baseten ---
from langchain.agents import create_agent

agent = create_agent(model="baseten:zai-org/GLM-5.2", tools=tools)

# --- Ollama ---
from langchain.agents import create_agent

agent = create_agent(model="ollama:north-mini-code-1.0", tools=tools)
